src/Altinn.App.PlatformServices/Interface/IAltinnApp.cs

```diff
@@ -133,6 +133,7 @@ public interface IAltinnApp
         /// <param name="dataTypeId">The data type id of the current layout.</param>
         /// <param name="formData">The form data.</param>
         /// <returns> The pages in sorted order.</returns>
+        [Obsolete("This method is deprecated. Use transient dependency of IStatefulPageOrder insted: https://docs.altinn.studio/app/development/ux/pages/tracks/")]
```

Comment:
My turn to do some nit picking - the Obsolete attribute has another signature where you explicitly set the url to the documentation see https://docs.microsoft.com/en-us/dotnet/api/system.obsoleteattribute?view=net-6.0